neural_compressor/torch/algorithms/weight_only/gptq.py

```diff
@@ -758,14 +755,16 @@ def find_params(self, x, weight=False):
         self.maxq = self.maxq.to(dev)
         # NF4 FP4
         if self.wdtype != "int":
-            from .rtn import quant_weight
+            from .utility import quant_tensor

```

Comment:
Please use the absolute path.

Reply:
Hi @Kaihui-intel , here we should use relative path. 
First, The utility in weight_only is pure for weight_only, relative path is more clean.
Second, the `algorithm` folder is a separate folder and allows usage without import INC, relative path makes it flexible.

Reply:
Please keep relative path for utility which is pure for its current folder.